### PR TITLE
created an create-account api

### PR DESCRIPTION
**Pull request by GagandeepKumar92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# accountsmgr
+# accounts-mgr
```
Comment (sewak2610) [inline README.md]: revert back.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 
-This is the accounts manager.
+// This is dummy text
```
Comment (sewak2610) [inline README.md]: revert

File: api/handlers/convertors.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+import (
+	"accountsmgr/domain"
+	"accountsmgr/gen/models"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+)
+
+func asUserResponse(d *domain.Account) *models.Account {
```
Comment (sewak2610) [inline api/handlers/convertors.go]: why user????

File: cmd/accountsmgr-server/main.go [hunk 4]
```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	AccountsMgr "accountsmgr"
```
Comment (sewak2610) [inline cmd/accountsmgr-server/main.go]: why is this alias needed?

File: cmd/accountsmgr-server/main.go [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+	// load embedded swagger file
+	swaggerSpec, err := loads.Analyzed(restapi.SwaggerJSON, "")
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// create new service API
+	api := operations.NewAccountsmgrAPI(swaggerSpec)
+
+	v := AccountsMgr.NewRunTime("Gagan")
```
Comment (sewak2610) [inline cmd/accountsmgr-server/main.go]: Gagan????
is this your application name???

File: db/mongo/client.go [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+	dbc *mongo.Database
+}
+
+func (c *client) CreateAccount(user *domain.Account) (string, error) {
+
+	fmt.Println("In Add User 1")
+	ctx, _ := context.WithTimeout(context.Background(), 5*time.Second)
+
+	_, err := c.dbc.Collection("accountsmgr").InsertOne(ctx, bson.D{
+		{Key: "_id", Value: user.ID},
```
Comment (sewak2610) [inline db/mongo/client.go]: need a change here.
how about user_id, user_name

File: swagger/swagger.yml [hunk 7]
```diff
@@ -0,0 +1,71 @@
+consumes:
+- application/json
+definitions:
+  account:
+    type: object
+    required:
+      - name
+    properties:
+      id:
```
Comment (sewak2610) [inline swagger/swagger.yml:43]: missing title, description and example .

File: swagger/swagger.yml [hunk 8]
```diff
@@ -0,0 +1,71 @@
+consumes:
+- application/json
+definitions:
+  account:
+    type: object
+    required:
+      - name
+    properties:
```
Comment (sewak2610) [inline swagger/swagger.yml:42]: missing user_id, user_name

File: swagger/swagger.yml [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+        type: integer
+        format: int64
+      message:
+        type: string
+  
+info:
+  title: Account Manager
+  version: 0.1.0
+paths:
+  /createaccount:
```
Comment (sewak2610) [inline swagger/swagger.yml]: /accounts

File: swagger/swagger.yml [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+            $ref: "#/definitions/error"
+        default:
+          description: error
+          schema:
+            $ref: "#/definitions/error"      
+produces:
+- application/json
+schemes:
+- http
+swagger: "2.0"
```
Comment (sewak2610) [inline swagger/swagger.yml]: shift to this top of the file.

File: swagger/swagger.yml [hunk 11]
```diff
@@ -0,0 +1,71 @@
+consumes:
+- application/json
+definitions:
```
Comment (sewak2610) [inline swagger/swagger.yml:36]: move definitions at bottom

**Review by sewak2610 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GagandeepKumar92) [reply README.md]: Done. Kindly Check.

File: README.md (hunk 2 above)
Comment (GagandeepKumar92) [reply README.md]: Done. Kindly Check.

File: api/handlers/convertors.go (hunk 3 above)
Comment (GagandeepKumar92) [reply api/handlers/convertors.go]: I had renamed it. But now removed, as I am not using it.

File: cmd/accountsmgr-server/main.go (hunk 4 above)
Comment (GagandeepKumar92) [reply cmd/accountsmgr-server/main.go]: I have removed it. Done. Kindly Check.

File: cmd/accountsmgr-server/main.go (hunk 5 above)
Comment (GagandeepKumar92) [reply cmd/accountsmgr-server/main.go]: Done. Kindly Check.

File: swagger/swagger.yml (hunk 7 above)
Comment (GagandeepKumar92) [reply swagger/swagger.yml:43]: Done. Kindly Check.

File: swagger/swagger.yml (hunk 8 above)
Comment (GagandeepKumar92) [reply swagger/swagger.yml:42]: Done. Kindly Check.

File: swagger/swagger.yml (hunk 9 above)
Comment (GagandeepKumar92) [reply swagger/swagger.yml]: Done. Kindly Check.

File: swagger/swagger.yml (hunk 10 above)
Comment (GagandeepKumar92) [reply swagger/swagger.yml]: Done. Kindly Check.

File: swagger/swagger.yml (hunk 11 above)
Comment (GagandeepKumar92) [reply swagger/swagger.yml:36]: Done. Kindly Check.

**Pull request merged**